fast_api_als/routers/submit_lead.py

```diff
@@ -148,16 +188,26 @@ async def submit(file: Request, apikey: APIKey = Depends(get_api_key)):
         response_body["status"] = "REJECTED"
         response_body["code"] = "16_LOW_SCORE"
 
+    s_time = int(time.time() * 1000.0)
+
     # verify the customer
     if response_body['status'] == 'ACCEPTED':
-        contact_verified = await new_verify_phone_and_email(email, phone)
+        try:
+            contact_verified = await new_verify_phone_and_email(email, phone)
+        except:
+            logging.info("/submit: fn submit: Verification of phone and email failed")
+        time_taken = (time.process_time() - s_time) * 1000
+        logging.info(f"/submit: fn submit : Verification of phone and email done in {time_taken} ms")
         if not contact_verified:
             response_body['status'] = 'REJECTED'
             response_body['code'] = '17_FAILED_CONTACT_VALIDATION'
 
     lead_uuid = str(uuid.uuid5(uuid.NAMESPACE_URL, email + phone + last_name + make + model))
-    item, path = create_quicksight_data(obj['adf']['prospect'], lead_uuid, response_body['status'],
+    try:
+        item, path = create_quicksight_data(obj['adf']['prospect'], lead_uuid, response_body['status'],
                                         response_body['code'], model_input)
+    except:
+        logging.error("/submit: fn submit: Unable to create quicksight data")
```

Comment:
if an exception occurred, how the code below would access the item & path?
won't they fail?